html/changelogs/JakeDaBoss.yml

```diff
@@ -33,4 +33,11 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - bugfix: "Corrected the accessible_z_levels list, which should correct a good deal of weirdness."
+  - bugfix: "Applied (hopefully) something that fixes all the stairs."
+  - bugfix: "Fixed unpowered APC in Paramedic's storage."
+  - bugfix: "Fixed a wrong wire in Upper Med."
+  - tweak: "Nerfed taser bolt's HALLOSS agony from 80 to 50."
```

Comment:
Players won't know what HALLOSS means. Changelogs should be phrased in a way that normal players can understand.